src/renderers/webgl/WebGLBackground.js

```diff
@@ -85,6 +86,12 @@ function WebGLBackground( renderer, state, objects, premultipliedAlpha ) {
 			boxMesh.material.uniforms.tCube.value = ( background.isWebGLRenderTargetCube ) ? background.texture : background;
 			boxMesh.material.uniforms.tFlip.value = ( background.isWebGLRenderTargetCube ) ? 1 : - 1;
 
+			if ( background !== previousBackground ) {
+
+				boxMesh.material.needsUpdate = true;
```

Comment:
What happens if the new texture has not loaded yet? Where is the new `texture.encoding` value assigned in the app? If it is in the texture loader callback, then that is a problem...